### DISCUSSION TITLE
a request context in the output schema

### DISCUSSION THREAD

**Discussion by jankrnavek** [vitalik/django-ninja#1143] [Q&A] [answered]
Hi,
Is it possible to get a request context (especially user) in the output schema?
I 'd like to use the user in some method ` def resolve_my_fields(obj)`

Thank,
H.

**Comment by vitalik** [comment, marked as answer]
@jankrnavek 

sure - https://django-ninja.dev/guides/response/?h=resolve_#accessing-extra-context

```Python
class Some(Schema):
    ...
    my_fields: list[User] = []

    @staticmethod
    def resolve_my_fields(obj, context):
        request = context["request"]
        user = request.user
        return user.friends.all()

```
